MultiSlider.js

```diff
@@ -240,20 +243,27 @@ class MultiSlider extends Component {
     } = this.state;
 
     if (this.state.disabled) {
-      console.log('------------------00001');
       return;
     }
-    console.log('------------------111111111');
-
 
     let nextX = this._getValue(gestureState.dx);
 
-    if (currentThumb === 'left' && (rightValue._value - nextX) > this.props.minSpace) {
-      leftValue.setValue(nextX);
-      this._fireChangeEvent('onLeftValueChange');
-    } else if (currentThumb === 'right' && (nextX - leftValue._value) > this.props.minSpace) {
-      rightValue.setValue(nextX);
-      this._fireChangeEvent('onRightValueChange');
+    if(this.props.allowSameValues) {
```

Comment:
Yeah, that was a bit ugly, I was in a hurry, sorry. Anyway the if statement was there because of the greater or greaterEqual operator, not the because of the spacing difference.